pyproject.toml

```diff
@@ -154,6 +154,14 @@ freeze = "pip freeze"
 test = "pytest ."
 test-cov = "pytest --cov=cosmos --cov-report=term-missing --cov-report=xml ."
 
+######################################
+# PYTEST
+######################################
+[tool.pytest.ini_options]
+filterwarnings = [
+    "ignore::DeprecationWarning",
```

Comment:
Why are we adding this..?

Reply:
There were lots of deprecation warnings from packages used by Airflow and other dependencies that grew to be quite noisy. Open to other ideas to make the test output more readable, but if you run them with and without this line you'll see the difference.